### PR TITLE
task 3 completed with test

### PR DESCRIPTION
**Pull request by bak-aleks**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+})
+
+describe('1. GET/api/topics', ()=>{
+    test('status 200, should respond with topics objects in array',()=>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response)=>{
+            expect(response.body.topics).toEqual(expect.any(Array));
+            expect(Object.keys(response.body.topics[0])).toEqual(
```
Comment (MKD01) [inline __tests__/app.test.js]: Are you only expecting a single object back from this endpoint? This assertion should test that all topics have the same keys and testing for what type the value of these are. Looking into .toMatchObject() will help in this case.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    if(db.end) db.end();
+})
+
+describe('1. GET/api/topics', ()=>{
+    test('status 200, should respond with topics objects in array',()=>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response)=>{
+            expect(response.body.topics).toEqual(expect.any(Array));
```
Comment (MKD01) [inline __tests__/app.test.js:21]: Testing that the array is at least greater than 0 is an assertion that should be added here as well to make sure an empty array does not pass this test.

File: app.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+const app = express()
+const {getTopics} = require('./controllers/news')
+
+app.use(express.json());
+app.get('/api/topics', getTopics);
+
+app.get('*', function(req, res){
```
Comment (MKD01) [inline app.js]: Has this been tested? I do like that this was added but please only add code that has a test so we know it does work.

**Review by MKD01 — CHANGES_REQUESTED** [review]
Please make these changes and send another PR request to reflect this.

**Review by dav3rid — APPROVED** [review]

**Pull request merged**